pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/ASTUserClass.java

```diff
@@ -67,4 +71,11 @@ public boolean hasSuppressWarningsAnnotationFor(Rule rule) {
     public ASTModifierNode getModifiers() {
         return getFirstChildOfType(ASTModifierNode.class);
     }
+
+
+    public String getSuperTypeName() {
```

Comment:
This should probably be named `getSuperClassName` since super-interfaces are also supertypes (if Java and Apex terminology is the same)
 
Besides, we should maybe be proactive and expose the names of super interfaces as well?